client/stylesheets/_toCat.scss

```diff
@@ -2875,8 +2879,91 @@ i:hover {
 
 .managerOfferButton ul li a:hover {
     background-color: #f73c37;
+} */
+
+.managerOfferButton {
+    margin-top: 20px !important;
+}
+
+.optionsCardItems li a:hover {
+	background-color: #d3d3d3 !important;
+}
+
+.optionsCardItems li a i {
+	margin-right: 5px !important;
+}
+
+.optionsListCardClick {
+	margin: 0 !important;
+	font-size: 0.9rem !important;
+}
+
+.optionsListCardClick ul {
+	position:relative !important;
+	font-weight:400 !important;
+}
+
+.optionsListCardClick ul li {
+	position:relative !important;
+	list-style:none !important;
+	cursor:pointer !important;
+}
+
+.optionsListCardClick ul li ul {
+	position:absolute !important;
+	left:0 !important;
+	right:0 !important;
+	border: 2px solid #f39d00 !important;
+	margin-top: -2px !important;
+}
+
+.optionsListCardClick ul .clicker {
+	position:relative !important;
+	background:orange !important;
+	color:#FFF !important;
+	text-align: center !important;
+	padding: 12px 8px !important;
+	border-radius: 4px !important;
+}
+
+.optionsListCardClick ul .clicker i {
+	color: #fff !important;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used<br>Line should be indented with spaces, not tabs